sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_LROPolling.md

```diff
@@ -0,0 +1,149 @@
+                        I have is the food didn't come fast enough. Overall I highly recommend it!";
+
+    var batchDocuments = new List<string> { document };
+
+    AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+    {
+        KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters(),
+        EntitiesTaskParameters = new EntitiesTaskParameters(),
+        PiiTaskParameters = new PiiTaskParameters(),
+        DisplayName = "AnalyzeOperationSample"
+    };
+
+    AnalyzeOperation operation = client.StartAnalyzeOperationBatch(batchDocuments, operationOptions);
+
+		TimeSpan pollingInterval = new TimeSpan(1000);
+
+		await operation.WaitForCompletionAsync(pollingInterval);
+
+    AnalyzeOperationResult resultCollection = operation.Value;
+
+    RecognizeEntitiesResultCollection entitiesResult = resultCollection.Tasks.EntityRecognitionTasks[0].Results;
+
+    ExtractKeyPhrasesResultCollection keyPhrasesResult = resultCollection.Tasks.KeyPhraseExtractionTasks[0].Results;
+
+    RecognizePiiEntitiesResultCollection piiResult = resultCollection.Tasks.EntityRecognitionPiiTasks[0].Results;
+```
+
+## Polling using `UpdateStatusAsync()`
+
+This method is for users who want to have intermittent code paths during the polling process.
+
+```C#
+    string document = @"We went to Contoso Steakhouse located at midtown NYC last week for a dinner party, 
+                        and we adore the spot! They provide marvelous food and they have a great menu. The
+                        chief cook happens to be the owner (I think his name is John Doe) and he is super 
+                        nice, coming out of the kitchen and greeted us all. We enjoyed very much dining in 
+                        the place! The Sirloin steak I ordered was tender and juicy, and the place was impeccably
+                        clean. You can even pre-order from their online menu at www.contososteakhouse.com, 
+                        call 312-555-0176 or send email to order@contososteakhouse.com! The only complaint 
+                        I have is the food didn't come fast enough. Overall I highly recommend it!";
+
+    var batchDocuments = new List<string> { document };
+
+    AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+    {
+        KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters(),
+        EntitiesTaskParameters = new EntitiesTaskParameters(),
+        PiiTaskParameters = new PiiTaskParameters(),
+        DisplayName = "AnalyzeOperationSample"
+    };
+
+    AnalyzeOperation operation = client.StartAnalyzeOperationBatch(batchDocuments, operationOptions);
+
+		TimeSpan pollingInterval = new TimeSpan(1000);
+
+		while (true)
+		{
+				await healthOperation.UpdateStatusAsync();
+				if (healthOperation.HasCompleted)
+				{
+						break;
+				}
+
+				await Task.Delay(pollingInterval).ConfigureAwait(false);
+		}
+
```

Comment:
format

Reply:
could you also add tips in case someone wants to use this code? i.e. consider having a max wait time, and add logging, etc?